restore-postgres-backup/action.yml

```diff
@@ -0,0 +1,74 @@
+name: Restore postgres from backup
+description: Restore an Azure postgres database from a daily backup
+inputs:
+  storage-account:
+    description: Name of the Azure storage account that contains the backup
+    type: string
+    required: true
+  app-name:
+    description: Name of the app deployment
+    type: string
+    required: true
+  resource-group:
+    description: Name of the Azure resource group
+    type: string
+    required: true
+  azure-credentials:
+    description: A JSON string containing service principle credentials.
+    required: true
+  environment:
+    description: App environment
+    required: true
+  backup-file:
+    description: Name of the backup file to restore
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: azure/login@v2
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - name: Setup postgres client
+      uses: DFE-Digital/github-actions/install-postgres-client@master
+      with:
+        version: 14
+
+    - name: Install kubectl
+      uses: DFE-Digital/github-actions/set-kubectl@master
+
+    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
+
+    - name: K8 setup
+      shell: bash
+      run: |
+        make ci ${{ inputs.environment }} get-cluster-credentials
```

Comment:
Relying on the make targets means the calling workflow must have checked out the code, and the targets must be there and behave the same way.
If they follow the template, it should be the case. But it may be safer and more generic to avoid it and run the commands without make.

Reply:
get-cluster-credentials reads some variables from the env.tfvars.json/global_config (like cluster & cluster_short) which would have to be another input? 
install-konduit doesn't need anything, so the commands could be substituted without anything else required.

Reply:
I think we should pass the cluster name and RG explicitly

Reply:
removed dependency on make